airflow/exceptions.py

```diff
@@ -30,8 +30,8 @@
 
 class AirflowException(Exception):
     """
-    Base class for all Airflow's errors.
-    Each custom exception should be derived from this class
+    Base class for all Airflow's errors
+    Each custom exception should be derived from this class.
     """
```

Comment:
PEP 257 recommends something like this:


```python
"""Base class for all Airflow's errors.

Each custom exception should be derived from this class.
"""
```

Reply:
Yeah I was thinking about what to do here because all of those classes are sort of wrong in how the docstring is formatted.

Reply:
There are a ton of weirdly formatted docstrings in Airflow unfortunately. But we can make the count one less since we are editing this anyway.

Reply:
Fixed in #20811.